WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductFromImage/AddProductFromImageView.swift

```diff
@@ -87,6 +87,14 @@ struct AddProductFromImageView: View {
                 }
                 .buttonStyle(LinkButtonStyle())
             }
+            ToolbarItem(placement: .cancellationAction) {
+                Button(action: {
+                    completion(nil)
+                }, label: {
+                    Image(systemName: "xmark")
```

Comment:
Nit: Can we use the "Cancel" title to be consistent with other screens like the Order creation form?

Reply:
Makes sense, updated in https://github.com/woocommerce/woocommerce-ios/pull/10237/commits/f7557ddd306b2eb40d8549fd71177280166fc704 plus the screenshots in the PR description